docs/reference/setup/restart-cluster.asciidoc

```diff
@@ -173,7 +173,18 @@ the datafeeds from {kib} or with the <<ml-open-job,open jobs>> and
 
 include::{es-repo-dir}/setup/restart-cluster.asciidoc[tag=disable_shard_alloc]
 
-include::{es-repo-dir}/setup/restart-cluster.asciidoc[tag=stop_indexing]
```

Comment:
The `stop_indexing` tag seems unused after this - can we remove the tag markers?

Reply:
That tag (and the associated `include`) were removed in this change, so we're 👍

Reply:
I don't follow - the tag markers are still there AFAICT?

```
$ git show db9fb02267714bfe8e37d6abbf7520955a7b3f48:docs/reference/setup/restart-cluster.asciidoc | grep stop_indexing
// tag::stop_indexing[]
// end::stop_indexing[]
```

Reply:
@DaveCTurner, you are correct! Gosh, I was looking at the `include` and not the `tag`. I'll get that squared away and backported appropriately.